src/app/layout.tsx

```diff
@@ -1,29 +1,27 @@
-'use client'
-import Basic from '@components/layout/Basic'
+
+import Basic from '@/components/layout/Basic'
 import './globals.css'
 import type { Metadata } from 'next'
-import Login from '@components/layout/Login'
-import { useAuthStore } from '@/stores/useAuthStore'
 
 export const metadata: Metadata = {
   title: 'book-piseo',
   description: '',
 }
 
-export default function RootLayout({ children }: { children: React.ReactNode }) {
-  const { isLogin } = useAuthStore()
-
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode
+}) {
   return (
     <html lang="en">
-      <link
-        rel="stylesheet"
-        as="style"
-        crossOrigin=""
-        href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/variable/pretendardvariable-dynamic-subset.css"
-        // @ts-ignore
-        precedence="default"
-      />
-      <body className="">{!isLogin ? <Login /> : <Basic>{children}</Basic>}</body>
```

Comment:
다른 페이지 개발을 진행하려고 useAuthStore의 isLogin의 초기값을 true로 설정했는데 (isLogin이 true여야 진입가능)
그럴 경우 아래 에러가 나오고 있습니다. 이 에러는 별도로 고려하지 않아도 되는 부분일까요? 

```
Error: Hydration failed because the initial UI does not match what was rendered on the server.
See more info here: https://nextjs.org/docs/messages/react-hydration-error
```

Reply:
오..이것은 제가 고려하지 못한 부분인 거 같아요! 감사합니다. 내일 수정해서 다시 pr 요청 드리겠습니다!